### PR TITLE
[Done] timeseriesexport is now async and downloads xlsx.

### PR DESCRIPTION
**Pull request by RoelvandenBerg**

### CODE REVIEW

File: app/components/timeseries/timeseries-directive.js [hunk 1]
```diff
@@ -128,14 +114,31 @@ angular.module('timeseries')
...
+            $http.get('/api/v2/timeseries/data/', {
+                params: {
+                  uuid: scope.timeseries.selected.uuid,
+                  start: Math.round(scope.timeState.start),
+                  end: Math.round(scope.timeState.end),
+                  format: 'xlsx',
+                  async: 'true'
+                }
+              }).then(function(response) {
+                console.log(response);
```
Comment (gijs) [inline app/components/timeseries/timeseries-directive.js]: console.log() mag weg

**Comment by gijs** [comment]
Eén console.log() weg en dan merge ik 'm

**Comment by RoelvandenBerg** [comment]
oei, die console.logjes zijn wel slordig...

**Comment by RoelvandenBerg** [comment]
Done

File: app/components/timeseries/timeseries-directive.js [hunk 2]
```diff
@@ -128,14 +114,30 @@ angular.module('timeseries')
...
+                  uuid: scope.timeseries.selected.uuid,
+                  start: Math.round(scope.timeState.start),
+                  end: Math.round(scope.timeState.end),
+                  format: 'xlsx',
+                  async: 'true'
+                }
+              }).then(function(response) {
+                notie.alert(
+                  4,
+                  "Downloading timeseries " + scope.timeseries.selected.name,
```
Comment (ernstkui) [inline app/components/timeseries/timeseries-directive.js]: This needs to be translated

Comment (ernstkui) [inline app/components/timeseries/timeseries-directive.js]: And, is this informative enough? It says dowloading timeseries, but users won't get a download box like they are used to.

Comment (ernstkui) [inline app/components/timeseries/timeseries-directive.js]: Maybe it should say: _preparing your xlsx_ and after a timeout _you will be notified when ready_ ofzo...

Comment (RoelvandenBerg) [inline app/components/timeseries/timeseries-directive.js]: I improved it. Still, when things are very slow the notie is also slow. After some thought I figured this is actually something you want. AFK explanation is available, just ask.

**Pull request merged**